source/core/replica-set-rollbacks.txt

```diff
@@ -100,4 +100,4 @@ the set by deleting the content of the :setting:`~storage.dbPath` directory for
 the member that requires a larger rollback.
 
 .. seealso:: :doc:`/core/replica-set-high-availability` and
-   :doc:`/core/replica-set-elections`.
+   :doc:`/core/replica-set-high-availability`.
```

Comment:
If I'm reading this correctly, it seems to say "see also replica-set-high-availability and replica-set-high-availability" which seems redundant.